CHANGELOG.md

```diff
@@ -3,7 +3,14 @@
 ### 0.11.0
 
 * Updates the Jackson dependency version to [2.9.10](https://github.com/FasterXML/jackson/wiki/Jackson-Release-2.9#patches)
-to address three security vulnerabilities in Jackson:
+to address three security vulnerabilities in Jackson.
+* Moves JSON Serializer/Deserializer implementations to a different package name.
+  - `io.jsonwebtoken.io.JacksonSerializer` -> `io.jsonwebtoken.jackson.io.JacksonSerializer`
+  - `io.jsonwebtoken.io.JacksonDeserializer` -> `io.jsonwebtoken.jackson.io.JacksonDeserializer`
+  - `io.jsonwebtoken.io.OrgJsonSerializer` -> `io.jsonwebtoken.orgjson.io.OrgJsonSerializer`
+  - `io.jsonwebtoken.io.OrgJsonDeserializer` -> `io.jsonwebtoken.orgjson.io.OrgJsonDeserializer`
+
+  A backward compatibility modules has been created using the `deprecated` classifier (`io.jsonwebtoken:jjwt-jackson:0.11.0:deprecated` and `io.jsonwebtoken:jjwt-orjson:0.11.0:deprecated`), if you are compiling against these classes directly, otherwise you will be unaffected.
```

Comment:
I think we should have this as as section in big bold warning, e.g. 
```
#### Backwards Compatibility Warning

Due to this package move, if you are currently using one of the above 4 existing (pre 0.11.0) classes with `compile` scope, you must either
  # change your code to use the newer package classes (recommended), or 
  # change your build/dependency configuration to use the `deprecated` dependency classifier 
    to use the existing classes, as follows:
      TODO:  need to show examples for both Maven and Gradle

Note that if you choose the second option, the `deprecated` classifier will not be 
available in the upcoming 1.0 release, so the recommended option is to choose 
the first option when possible.
```

Reply:
@lhazlewood I only used Jackson as the Maven/Gradle example, I'm thinking that should be enough (given the inline Gradle example above for `orgjson`) ?

Good call on the H4, it helps make this section stand out!

Reply:
I meant to show it just like we do in the primary documentation since it won't be obvious to a lot of people how to use classifiers, i.e.:

![image](https://user-images.githubusercontent.com/121180/65795566-8e1a0980-e138-11e9-8750-de55003e9821.png)

(but only need one declaration each for maven and gradle - not all of them like shown in the screenshot

Reply:
Ah, you did that in the update - I missed it.  Thanks!

Reply:
It wasn't just you, it took me a second to find it after I pushed too  😄